### PR TITLE
Some clarifications + fix typos in Bickley example

### PR DESCRIPTION
**Pull request by navidcy**
There are always leftover typos :)

@francispoulin, I tried to use your repo to compute the most unstable mode for k=1 and I couldn't do it...
Could you tell me what the actual growth rate for `k = 1` is? I remember 0.138? Am I right? We should report the theoretical growth rate for `k = 1` that fits this domain (and not the overall maximum growth rate) and then compare that with the best-fit prediction. What do you think?

### CODE REVIEW

File: examples/Bickley_jet_shallow_water.jl [hunk 1]
```diff
@@ -76,43 +69,51 @@ model = ShallowWaterModel(
...
+# \end{aligned}
 # ```
-# 
-# The vorticity of the background state is ``Ω = - \partial_y \overline{u} = 2 U \mathrm{sech}^2(y) \tanh(y)``.
 #
-# Linear stability theory predicts that for the particular parameters that we consider here,
-# the growth rate for the maximum growth rate should be ``0.14``.
+# The total height of the fluid is ``h = L_z + \eta``. Linear stability theory predicts that 
+# for the parameters we consider here, the growth rate for the most unstable mode that fits 
+# our domain is approximately ``0.14``.
```
Comment (navidcy) [inline examples/Bickley_jet_shallow_water.jl:85]: @francispoulin is 0.14 the growth rate for  `k = 1`?

Comment (francispoulin) [reply examples/Bickley_jet_shallow_water.jl:85]: Yes, to two significant digits, it is `0.14`.

**Comment by francispoulin** [comment]
@navidcy thanks for trying to use my repo and sorry it didn't work out.  My next step is to make docs, which should help. 
In case you are curious, the code that produced this data is [here](https://github.com/francispoulin/Linear-Stability-Calculators/blob/main/ShallowWater/Julia/Cartesian/linear_stability_shallow_water.jl).  

As for your question, the maximum growth rate for `k=0.95` was `\sigma=0.1397`, whereas for `k=1.0` it is `\sigma=0.139`.  Both are pretty much `0.14` so we do report the actual value.

File: examples/Bickley_jet_shallow_water.jl (hunk 1 above)
Comment (navidcy) [reply examples/Bickley_jet_shallow_water.jl:85]: is it 0.138 if we keep 3 significant? because that's really very close to the computed one

Comment (francispoulin) [reply examples/Bickley_jet_shallow_water.jl:85]: More like `0.139`

File: examples/Bickley_jet_shallow_water.jl [hunk 2]
```diff
@@ -76,43 +69,51 @@ model = ShallowWaterModel(
 
 # ## Background state and perturbation
 #
-# The background velocity ``\overline{u}`` and free-surface ``\overline{\eta}`` are chosen to 
-# represent a geostrophically balanced Bickely jet with maximum speed of ``U`` and 
-# maximum free-surface deformation of ``Δη``, i.e.,
+# The background velocity ``ū`` and free-surface ``η̄`` are chosen to represent a
```
Comment (francispoulin) [inline examples/Bickley_jet_shallow_water.jl:72]: I agree that using the short version is better and thanks for suggesting this. But does the overline on the `\eta` look too far to the right?  I guess it's how it looks in the html that matters.

File: examples/Bickley_jet_shallow_water.jl [hunk 3]
```diff
@@ -76,43 +69,51 @@ model = ShallowWaterModel(
...
 # 
-# We specify `Ω` to be the background vorticity of the jet in the absence of any perturbations.
-# The initial conditions include a small-ampitude perturbation that decays away from the center
-# of the jet.
-
-  Ω(x, y, z) = 2 * U * sech(y)^2 * tanh(y)
- uⁱ(x, y, z) =   U * sech(y)^2 + ϵ * exp(- y^2 ) * randn()
- ηⁱ(x, y, z) = -Δη * tanh(y)
- hⁱ(x, y, z) = model.grid.Lz + ηⁱ(x, y, z)
+# We also specify `ω̄` as the vorticity of the background state,
```
Comment (francispoulin) [inline examples/Bickley_jet_shallow_water.jl:87]: Again, overline looks weird but does not matter here as it does in the final product

File: examples/Bickley_jet_shallow_water.jl [hunk 4]
```diff
@@ -76,43 +69,51 @@ model = ShallowWaterModel(
...
 # We compute the total vorticity and the perturbation vorticity.
 
 uh, vh, h = model.solution
+
         u = ComputedField(uh / h)
         v = ComputedField(vh / h)
-        v = ComputedField(v)
-        ω = ComputedField(∂x(vh/h) - ∂y(uh/h))
-   ω_pert = ComputedField(ω - Ω)
+        ω = ComputedField(∂x(v) - ∂y(u))
```
Comment (francispoulin) [inline examples/Bickley_jet_shallow_water.jl:115]: Glad you got this to work and sorry that I didn't

File: examples/Bickley_jet_shallow_water.jl [hunk 5]
```diff
@@ -76,43 +69,51 @@ model = ShallowWaterModel(
...
+   ω_pert = ComputedField(ω - ω̄)
```
Comment (francispoulin) [inline examples/Bickley_jet_shallow_water.jl:116]: Much better

File: examples/Bickley_jet_shallow_water.jl (hunk 2 above)
Comment (navidcy) [reply examples/Bickley_jet_shallow_water.jl:72]: hm... it looks fine on my font :)

Screen Shot 2021-03-15 at 8 07 34 am

File: examples/Bickley_jet_shallow_water.jl [hunk 6]
```diff
@@ -246,23 +245,24 @@ I = 6000:7000
...
+        label = "best fit")
             
-# We can compute the slope of the curve on a log scale, which approximates the growth rate
-# of the simulation. This should be close to the theoretical prediction.
+# We can compute the slope of the curve on a logarithmic scale, which approximates the growth 
+# rate of the simulation. This should be close to the theoretical prediction.
 
-println("Growth rate in the simulation is approximated to be ", linear_fit_polynomial[1], ",\n",
+println("Numerical growth rate is approximated to be ", linear_fit_polynomial[1], ",\n",
         "which is close to the theoretical value of 0.14.")
```
Comment (francispoulin) [inline examples/Bickley_jet_shallow_water.jl]: Some could say is equal to the theoretical value. Never found that before but here we do

Comment (navidcy) [reply examples/Bickley_jet_shallow_water.jl]: Shall I change it to `very close`?

File: examples/Bickley_jet_shallow_water.jl (hunk 2 above)
Comment (francispoulin) [reply examples/Bickley_jet_shallow_water.jl:72]: Mine looks a bit odd but again, this is only the code so don't think this is a problem, just wanted to mention it.
Screenshot from 2021-03-14 17-12-54

File: examples/Bickley_jet_shallow_water.jl (hunk 6 above)
Comment (francispoulin) [reply examples/Bickley_jet_shallow_water.jl]: Sure

File: examples/Bickley_jet_shallow_water.jl (hunk 2 above)
Comment (navidcy) [reply examples/Bickley_jet_shallow_water.jl:72]: Have a look at this https://cormullion.github.io/pages/2020-07-26-JuliaMono/

Comment (francispoulin) [reply examples/Bickley_jet_shallow_water.jl:72]: Thanks and will try and get that working on my computer

**Review by francispoulin — APPROVED** [review]
Looks even better thanks to your hard work!

**Pull request merged**